### PR TITLE
Add/Fix ArcRotateCamera Pinch to Zoom with new NaturalPinchZoom type

### PR DESCRIPTION
**Pull request by rwedoff**
## What Changed
The current pinch to zoom functionality for the ArcRotateCamera is not generic to all models without setting various parameters. Therefore, I added a new pinch type (to support backwards compatibility) that uses the distance of the camera in relation to the pinch distance. As a result, this is a more natural way to zoom in on a model and it generic to any model, without breaking the current pinch to zoom paradigm. 

## Usage
var camera = new BABYLON.ArcRotateCamera(...);
camera.useNaturalPinchZoom = true;
PinchZoom

### CODE REVIEW

File: src/Cameras/Inputs/arcRotateCameraPointersInput.ts [hunk 1]
```diff
@@ -58,6 +58,14 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
     @serialize()
     public pinchDeltaPercentage = 0;
 
+    /**
+     * useNaturalPinchZoom will be used instead of pinchPrecision if set to true.
```
Comment (AdamMitchell-ms) [inline src/Cameras/Inputs/arcRotateCameraPointersInput.ts]: I'd rephrase this comment (and others in this review) to be something like this:

When useNaturalPinchZoom is true, multitouch zoom will zoom in such that any object in the plane at the camera's target point will scale perfectly with finger motion.  Overrides pinchDeltaPercentage and pinchPrecision.

File: src/Cameras/Inputs/arcRotateCameraPointersInput.ts [hunk 2]
```diff
@@ -139,7 +147,11 @@ export class ArcRotateCameraPointersInput extends BaseCameraPointersInput {
                 this.camera.inertialRadiusOffset +=
                     (pinchSquaredDistance - previousPinchSquaredDistance) * 0.001 *
                     this.camera.radius * this.pinchDeltaPercentage;
-            } else {
+            } else if (this.useNaturalPinchZoom) {
```
Comment (AdamMitchell-ms) [inline src/Cameras/Inputs/arcRotateCameraPointersInput.ts]: This should override pinchDeltaPercentage, if set.  So put it as the first IF in this set of blocks.

Comment (rwedoff) [reply src/Cameras/Inputs/arcRotateCameraPointersInput.ts]: Good point, will fix.

**Comment by deltakosh** [comment]
Excellent! Thanks a lot!
